clients/cobol-lsp-vscode-extension/src/__tests__/commands/FetchCopybookCommandTest.spec.ts

```diff
@@ -21,25 +21,28 @@ import { TelemetryService } from "../../services/reporter/TelemetryService";
 import { Utils } from "../../services/util/Utils";
 
 jest.mock("../../services/reporter/TelemetryService");
-jest.mock("../../services/copybook/CopybookDownloadService");
-Utils.getZoweExplorerAPI = jest.fn();
+Utils.getZoweExplorerAPI = jest.fn().mockReturnValue({});
 
-const copybookDownloadService: CopybookDownloadService =
-  new CopybookDownloadService("/storagePath");
 const copybook: string = "cobyBookTest";
 const progName: string = "progNameTest";
 
-test("Test fetchCopybookCommand calls telementry services and coprbook download service with proper input", () => {
+test("Test fetchCopybookCommand calls telementry services and copybook download service with proper input", async () => {
+  Utils.getZoweExplorerAPI = jest.fn().mockReturnValue({});
+  const copybookDownloadService: CopybookDownloadService =
+    new CopybookDownloadService(
+      "./storage-path",
+      await Utils.getZoweExplorerAPI(),
```

Comment:
at this point you can just pass in `{} as any as IApiRegisterClient`

Reply:
updated